jenkins/catapult/installDepsLocal.py

```diff
@@ -96,6 +96,10 @@ def build_git_dependency(self, organization, project):
 			# https://devblogs.microsoft.com/cppblog/msvc-now-correctly-reports-__cplusplus/
 			cmake_options += ['-DCMAKE_CXX_FLAGS="/Zc:__cplusplus"', f'-DCMAKE_PREFIX_PATH={self.target_directory / organization}']
 
+			version = self.versions[f'{organization}_{project}']
+			if 'r3.10.0' <= version:
```

Comment:
this is just lexical compare, so it's not *completely* right, but probably good enough